pysand/erosion.py

```diff
@@ -37,9 +37,11 @@ def validate_inputs(**kwargs):
         if i in kwargs:
             if not isinstance(kwargs[i], (float, int, np.ndarray, pd.Series)) or np.isnan(kwargs[i]):
                 raise exc.FunctionInputFail('{} is not a number or pandas series'.format(i))
+            if not kwargs[i] >= 0:
+                return True
```

Comment:
we should throw a warning here as well, don't you think? If any erosion results throw np.nan it's not possible to see why..

Reply:
Yes, throwing a warning message on this is a good idea.
What do you mean by not possible to see? For negative input the result is nan